PowerScout/MasterControllers/DataTransferViewController.swift

```diff
@@ -42,12 +44,17 @@ class DataTransferViewController: UIViewController, ServiceStoreDelegate {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
+        if let appDelegate = UIApplication.shared.delegate as? AppDelegate {
```

Comment:
This usage of `UIApplication.shared.delegate` references the UIApplication singleton. I'm not sure what behavior this will introduce during testing, but it is null safeguarded and a new service store will be created if this doesn't exist. 

Once progress has started on #10, this may change. For now, this is okay.